lib/event.rb

```diff
@@ -0,0 +1,12 @@
+class Event
+  attr_reader :name, :crafts, :person
+  def initialize(name, crafts, person)
+    @name = name
+    @crafts = [crafts]
+    @person= [person]
+  end
+
+  def attendee_names
+    [person.name]
+  end
+end
```

Comment:
Your naming choices are working against you here. I would stick with what is defined in the interaction pattern. An even has attendees with is an array of person objects. You have referred to this as person (which insinuates that it is a single person object). This makes it harder to understand what data you are working with at any given point in time.

Since you wrapped the value of person on line 6 in square brackets the return value of person is an array within an array instead of a person object. The array doesn't have a `name` attribute. The person object has the name attribute. Additionally as mentioned above the person attribute is actually an array of person objects so you will want to use an enumerable such as `map` to iterate over the object in the array and then call person.name with the enumerable block.
For example:

```
person.map do |person|
  person.name
end

```